game/games/sortit/ui/player_select.gd

```diff
@@ -50,7 +55,12 @@ func get_get_input() -> bool:
 
 
 func _ready():
+	if Engine.is_editor_hint():
```

Comment:
I haven't looked over the whole node, but is there any specific reason this is actually a `tool` script in the first place?

Reply:
yes, also this was a tool before this PR and since that was merged (Accepted by more than 75%), I see no reason to change this.